### ISSUE TITLE
Yosys outdated

### ISSUE THREAD

**Issue by amarvir02** [f-of-e/f-of-e-tools#8]
When compiling Yosys, the process would crash due to errors regarding not including certain header files, I'm assuming used by newer compilers or something?
Some errors:
> passes/sat/freduce.cc:531:27: error: no member named 'numeric_limits' in namespace 'std'
                                out_depth[idx] = std::numeric_limits<int>::max();
                                                 ~~~~~^
> passes/sat/freduce.cc:531:45: error: expected '(' for function-style cast or type construction
                                out_depth[idx] = std::numeric_limits<int>::max();
                                                                     ~~~^
> passes/sat/freduce.cc:531:46: error: no matching function for call to 'max'
                                out_depth[idx] = std::numeric_limits<int>::max();
![image](https://github.com/f-of-e/f-of-e-tools/assets/72991909/091876f3-450f-4f1d-9183-1600d55a771b)
On WSL w/ Ubuntu 20.04, GCC11, Clang13

TLDR is yosys (and prob everything else) needs to be updated from not 3yrs ago
